src/main/java/com/qualfacul/hades/conversation/MessageDTO.java

```diff
@@ -0,0 +1,47 @@
+package com.qualfacul.hades.conversation;
+
+import java.util.Calendar;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.qualfacul.hades.serialization.FullDateCalendarSerializer;
+
+
+public class MessageDTO {
+	
+	private Long id;
+	
+	@JsonSerialize(using = FullDateCalendarSerializer.class)
+	private Calendar sentAt;
```

Comment:
que tal usar um `LocalDateTime`?
se for usar, precisa criar um converter, vide `LocalDateAttributeConverter`